web/src/server_events_dispatch.js

```diff
@@ -588,22 +589,26 @@ export function dispatch_normal_event(event) {
                     const user_ids = people.validate_user_ids(event.user_ids);
 
                     peer_data.bulk_add_subscribers({stream_ids, user_ids});
-                    stream_events.process_subscriber_update(stream_ids);
+                    stream_events.process_subscriber_update(user_ids, stream_ids);
                     break;
                 }
                 case "peer_remove": {
                     const stream_ids = sub_store.validate_stream_ids(event.stream_ids);
                     const user_ids = people.validate_user_ids(event.user_ids);
 
                     peer_data.bulk_remove_subscribers({stream_ids, user_ids});
-                    stream_events.process_subscriber_update(stream_ids);
+                    stream_events.process_subscriber_update(user_ids, stream_ids);
                     break;
                 }
                 case "remove":
                     for (const rec of event.subscriptions) {
                         const sub = sub_store.get(rec.stream_id);
                         stream_events.mark_unsubscribed(sub);
                     }
+                    if (overlays.get_user_id_if_user_profile_modal_open()) {
+                        const modalUserId = overlays.get_user_id_if_user_profile_modal_open();
```

Comment:
`modalUserId` -> `user_id` in the other commits too.

Reply:
Hey @amanagr , I am not sure what you mean over here, should I replace `modaUserId` with `user_id` or correct the commit description.

Reply:
> , should I replace modaUserId with user_id 

Yes.

Reply:
Thanks for the suggestion, I have updated the PR, could you please review again, thanks :)